osu.Game/Skinning/SkinInfo.cs

```diff
@@ -47,7 +47,17 @@ public virtual Skin CreateInstance(IStorageResourceProvider resources)
                 ? typeof(LegacySkin)
                 : Type.GetType(InstantiationInfo).AsNonNull();
 
-            return (Skin)Activator.CreateInstance(type, this, resources);
+            try
+            {
+                return (Skin)Activator.CreateInstance(type, this, resources);
+            }
+            catch
+            {
+                // This defaults to triangles to catch the case where a user has a modified triangles skin.
+                // If we ever add more default skins in the future this will need some kind of proper migration rather than
+                // a single catch.
+                return new TrianglesSkin(this, resources);
+            }
```

Comment:
~~Is this intended to be included in this PR? I thought it was just a code quality change regarding renames.~~ Never mind, users will now have a `DefaultSkin` type which no longer exists in this project, so this logic will fall back to `TrianglesSkin` in attempt to catch it. Wasn't quite clear by the comment.